master/master.c

```diff
@@ -2839,22 +2839,7 @@ static void reread_conf(struct timeval now)
 
 static void check_undermanned(struct service *s, int si, int wdi)
 {
-    if (s->exec /* enabled */ &&
-        (s->nactive < s->max_workers) &&
-        (s->ready_workers < s->desired_workers))
-    {
-        /* bring us up to desired_workers */
-        int j = s->desired_workers - s->ready_workers;
-
-        if (verbose) {
-            syslog(LOG_DEBUG, "service %s/%s needs %d more ready workers",
-                   s->name, s->familyname, j);
-        }
-
-        while (j-- > 0) {
-            spawn_service(s, si, wdi);
-        }
-    } else if (s->exec
+    if (s->exec
                 && s->babysit
                 && s->nactive == 0) {
         if (s->nreadyfails >= MAX_READY_FAILS) {
```

Comment:
Style nit:

```suggestion
    if (s->exec
        && s->babysit
        && s->nactive == 0)
    {
        if (s->nreadyfails >= MAX_READY_FAILS) {
```

* align the terms in the condition
* since this puts the condition terms at the same indent level as the block body (due to `if (` being four chars), shunt the `{` to its own line to make them visually distinct